.devcontainer/devcontainer.json

```
@@ -0,0 +1,14 @@
+// See https://aka.ms/vscode-remote/devcontainer.json for format details.
+{
+	"name": "C# (.NET Core 2.1) and PowerShell on Debian 9",
+	"dockerFile": "Dockerfile",
+
+	"postCreateCommand": "dotnet restore",
+
+	"extensions": [
+        "ms-vscode.csharp",
+        "ms-vscode.powershell",
```

```suggestion
        "ms-vscode.powershell-preview",
```
The developer of PSES should probably always use the latest and greatest PSES so we should use the Preview.